scripts/color-sorting.js

```diff
@@ -51,23 +51,23 @@ const rgbToHsl = (rgb) => {
   s = delta === 0 ? 0 : delta / (1 - Math.abs(2 * l - 1));
 
   // Multiply l and s by 100 to get the value in percent, rather than [0,1]
-  s = +(s * 100);
-  l = +(l * 100);
-  return { h, s, l };
+  s = Number(s * 100);
+  l = Number(l * 100);
```

Comment:
I don't understand this change. Is `Number` really required here?

Reply:
https://eslint.org/docs/latest/rules/no-implicit-coercion

Then `+()` is not necessary here either.

Reply:
Yeah, you can remove the `+()` coercion, which is totally useless code when dealing with numbers.